local_settings_example.py

```diff
@@ -1,2 +1,2 @@
 USER='myusername'
-PASSWORD='mypassword'
+PASSWORD='mypassword'
```

Comment:
@CharlesNepote was USER and PASSWORD intended for the database (but not used ?)

Reply:
I think this is for the Authentication service in OpenFoodFacts. So, the USER and PASSWORD are OpenFoodFacts credentials.